day_two_guide/topics/proc_restoring-etcd.adoc

```diff
@@ -273,14 +273,18 @@ Before restoring etcd on a static pod:
 * `etcdctl` binaries must be available or, in containerized installations,
 the `rhel7/etcd` container must be available.
 +
-You can obtain etcd by running the following commands:
+You can install the `etcdctl` binary with the etcd package by running the following commands:
 +
 ----
-$ git clone https://github.com/coreos/etcd.git
-$ cd etcd
-$ ./build
+# yum install etcd
 ----
 
+The package also installs the systemd service. Disable and mask the service as it shouldn't run as a systemd service when etcd runs in static pod.
```

Comment:
```suggestion
The package also installs the systemd service. Disable and mask the service so that it does not run as a systemd service when etcd runs in static pod. By disabling and masking the service, you ensure that you do not accidentally start it and prevent it from automatically restarting when you reboot the system.
```

@vwalek, will you make an update similar to this one? It's better to explain than to just say that you shouldn't do a thing.

Reply:
@kalexand-rh done :) Please review